packages/atomic/src/components/result-template-components-v1/atomic-result-sections/atomic-result-section-visual.tsx

```diff
@@ -2,19 +2,18 @@ import {Element, Component} from '@stencil/core';
 import {containsVisualElement} from '../../../utils/utils';
 
 /**
- * The visual section helps the information seeker understand information about
- * the item visually, for example in Commerce an image is a great descriptive
- * for some category of products, an icon can help understand what type an item
- * is or an avatar understand whom it is related to.
+ * This section provides visual information about the item.
+ * For example, in Commerce, an image is a great shorthand for a product category.
```

Comment:
I feel like the previous description meant that an image is descriptive of a result/product, similar to the proverb "An image is worth a thousand words". The new one implies to me that we should put an image shared between all results/products in a category.